bukkit/src/main/java/co/aikar/commands/BukkitCommandContexts.java

```diff
@@ -241,6 +241,20 @@ OnlinePlayer getOnlinePlayer(BukkitCommandIssuer issuer, String lookup, boolean
         Player player = ACFBukkitUtil.findPlayerSmart(issuer, lookup);
         //noinspection Duplicates
         if (player == null) {
+            if (issuer.getManager() instanceof BukkitCommandManager) {
```

Comment:
this is already a bukkit context, this isn't really needed. 

this.manager should already exists.

Reply:
this.manager returns a `CommandManager` which does not have the version fields